apps/dashboard/app/views/files/_shell.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="shell-wrapper btn-group dropright">
+  <%= link_to OodAppkit.shell.url(path: @path.to_s).to_s, id: 'open-in-terminal-btn', class: 'btn btn-outline-dark btn-sm disabled', target: '_blank' do %>
+    <i class="fas fa-terminal" aria-hidden="true"></i>
+    Open in Terminal
+  <% end %>
+
+  <% if OodAppkit.clusters.count > 0 %>
+    <button type="button" class="btn btn-sm btn-outline-dark dropdown-toggle dropdown-toggle-split" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+    <ul class="dropdown-menu" x-placement="right-start" id="shell-dropdown-items">
+      <%
+      clusters.each{ |cluster|
+        tag.a cluster.title, href: 'link', class: 'dropdown-item', role: 'menuitem'
```

Comment:
`href: 'link'` instead of the shell link

maybe you should have a method `OodApp.shell_links` that can be used here and in the `OodApp#links`

Reply:
then you can use the `OodAppLink#url` for the 'link'

I guess we should also use the word "Shell" instead of "Terminal" since thats what we use elsewhere.